tests/integration/components/sl-button-test.js

```diff
@@ -0,0 +1,206 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import sinon from 'sinon';
+
+moduleForComponent( 'sl-button', 'Integration | Component | sl button', {
+    integration: true
+});
+
+const mockStreamService = {
+    send() {}
+};
+
+const template = hbs`
+    {{#sl-button}}
+        Default Text
+    {{/sl-button}}
+`;
+
+/**
+ * While it appears that core Ember functionality is being tested this test is
+ * ensuring that the implied contract about which DOM element is rendered is
+ * adhered to.
+ */
+
+test( 'Button tag is rendered', function( assert ) {
+    this.render( template );
+
+    assert.ok(
+        this.$( '>:first-child' ).is( ':button' ),
+        'button tag was rendered'
+    );
+});
+
+test( 'Default action is triggered when element is clicked event when clicked', function( assert ) {
```

Comment:
Does the title of this test seem off a little or am I not understanding the meaning?